app/views/clients/users/index.haml

```diff
@@ -1,4 +1,4 @@
-- title = "Assigned Relationships for #{@client.pii_provider(user: current_user).full_name}"
+- title = "Assigned Relationships"
```

Comment:
Should this have an `H1` like the other pages?

Reply:
I can add one, I was trying not change the existing body html

Reply:
Should be fine for now then.  Let's leave it as is

Reply:
thanks I'll leave it to limit scope. It looks like none of the dash tabs have an h1